frontend/src/middleware/collection.ts

```diff
@@ -78,6 +83,14 @@ export const collectionMiddleware = defineNuxtRouteMiddleware(async (to) => {
     })
   }
 
+  const s = useSiteConfig()
+
+  if (s.indexable && collectionParams.collection === "source") {
```

Comment:
```suggestion
  const siteConfig = useSiteConfig()

  if (siteConfig.indexable && collectionParams.collection === "source") {
```

Super nit: from what I've seen we usually favor explicit verbosity over brevity in most of the other frontend code.

Reply:
Whoops, definitely, I wasn't sure `useSiteConfig` was going to work and meant to change it if it did! Thanks for catching that.